evident/tests/test_diversity_handler.py

```diff
@@ -0,0 +1,237 @@
+import os
```

Comment:
Fine to leave it like this but out of curiosity, why not use a different test_*_.py file for each object?

Reply:
Probably would have been better to do that.